report/thermodesorption-spectra/ogorodnikova-tds.ipynb

```diff
@@ -0,0 +1,299 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Thermo-desorption spectrum\n",
+    "\n",
+    "This validation case is a thermo-desorption spectrum measurement perfomed by Ogorodnikova et al. {cite}`ogorodnikova_deuterium_2003`.\n",
+    "\n",
+    "Deuterium ions at 200 eV were implanted in a 0.5 mm thick sample of high purity tungsten foil (PCW).\n",
+    "\n",
+    "The ion beam was turned on for 400 s which corresponds to a fluence of $1.0 \\times 10^{22} \\ \\mathrm{D \\ m^{-2}}$\n",
+    "\n",
+    "The trapping parameters are taken from Table 1 for both the intrinsic traps and the extrinsic, ion-induced traps."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## FESTIM code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {
+    "tags": [
+     "hide-input",
+     "hide-output"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Defining initial values\n",
+      "Defining variational problem\n",
+      "Defining source terms\n",
+      "Defining boundary conditions\n",
+      "Time stepping...\n",
+      "0.3 %        1.4e+00 s    Ellapsed time so far: 0.2 s\r"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/komodo/anaconda3/envs/vv-festim-report-env/lib/python3.11/site-packages/festim/exports/derived_quantities/derived_quantities.py:52: DeprecationWarning: The derived_quantities attribute will be deprecated in a future release, please use festim.DerivedQuantities as a list instead\n",
+      "  warnings.warn(\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "100.0 %        5.0e+02 s    Ellapsed time so far: 10.3 s\n"
+     ]
+    }
+   ],
+   "source": [
+    "import festim as F\n",
+    "import numpy as np\n",
+    "import sympy as sp\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "flux_model = F.Simulation()\n",
+    "\n",
+    "vertices = np.concatenate([\n",
+    "    np.linspace(0, 30e-9, num=200),\n",
+    "    np.linspace(30e-9, 3e-6, num=300),\n",
+    "    np.linspace(3e-6, 20e-6, num=200),\n",
+    "])\n",
+    "\n",
+    "flux_model.mesh = F.MeshFromVertices(vertices)\n",
+    "# Material Setup, only W\n",
+    "#　Data from Table 1\n",
+    "tungsten = F.Material(\n",
+    "    id=1,\n",
+    "    D_0=4.1e-07,  # m2/s\n",
+    "    E_D=0.39,  # eV\n",
+    ")\n",
+    "\n",
+    "flux_model.materials = tungsten\n",
+    "import sympy as sp\n",
+    "\n",
+    "imp_fluence = 1e22\n",
+    "flux_strength = 2.5e19 # beam strength from paper\n",
```

Comment:
Instead of `flux_strength` we typically call this the incident flux.

Moreover, we need to have a discussion with @gabriele-ferrero and @ehodille to lift the uncertainties between implanted flux, incident flux, reflection coefficient, etc...